index.html

```diff
@@ -2128,6 +2291,220 @@ <h4>Bidirectional text</h4>
+        such a document.</p>
+      </section>
+
+      <section>
+        <h3>Not supporting features excluded by the content profile</h3>
+        <p>A processor that takes as its input a <a>DAPT document</a> that
+          contains vocabulary relating to features that it does support,
+          but where support for those features is excluded from the content profiles
+          to which the document claims conformance, SHOULD NOT implement
+          those features in the context of that document.</p>
+      </section>
+      <section>
+        <h3>Handling <code>&lt;div&gt;</code> and <code>&lt;p&gt;</code> elements</h3>
+        <p>[[TTML2]] allows <code>&lt;div&gt;</code> elements to contain any combination of
+          <code>&lt;div&gt;</code> elements and <code>&lt;p&gt;</code> elements.
+          The DAPT data model describes how each <a>Script Event</a> is represented by
+          a <code>&lt;div&gt;</code> element that contains zero or more <code>&lt;p&gt;</code> elements.
+          It also permits other intermediate <code>&lt;div&gt;</code> elements in the path between
+          the <code>&lt;body&gt;</code> element and those <a>Script Event</a> <code>&lt;div&gt;</code> elements.
+          In addition, attributes not corresponding to properties in the DAPT data model are permitted.</p>
+        <p>This gives rise to possibilities such as:</p>
+        <ul>
+          <li><code>&lt;div&gt;</code> elements that contain children that are not <a>Text</a> objects,
+            for example other <code>&lt;div&gt;</code> elements or <code>&lt;metadata&gt;</code> elements;</li>
+          <li><code>&lt;div&gt;</code> elements that do not have attributes required for <a>Script Events</a>, like <code>xml:id</code>;</li>
+          <li><code>&lt;div&gt;</code> elements that have attributes absent from the DAPT data model for <a>Script Events</a>,
+            like <code>xml:space</code>;</li>
+          <li><code>&lt;p&gt;</code> elements whose parent <code>&lt;div&gt;</code> element does not appear to be a <a>Script Event</a>.</li>
+        </ul>
+
+        <p>The following processing rules resolve these cases.</p>
+        <p>Rules for identifying <a>Script Events</a>:</p>
+        <ol>
+          <li>A <code>&lt;div&gt;</code> element that has no <code>&lt;div&gt;</code> element children
+            and includes the TTML representations of all the mandatory properties of a <a>Script Event</a>
+            MUST be mapped to a <a>Script Event</a>,
+            such as having a valid <code>xml:id</code>
+            representing the <a>Script Event Identifier</a>,
+            even if it also contains additional <a>unrecognised vocabulary</a>;</li>
+          <li>A <code>&lt;div&gt;</code> element that contains any <code>&lt;div&gt;</code> element children
+            MUST NOT be mapped to a <a>Script Event</a>;
+            the processor instead MUST iterate through those <code>&lt;div&gt;</code> element children
```

Comment:
Do we care about the order of reconstructed divs? Probably not. I mean a depth-first or breadth-first will produce a different order, but given that attribute computation will produce defined begin/end values, that should drive the order.

Reply:
We should care about document order, yes. A depth-first approach should preserve that - probably worth clarifying.

Reply:
Added in 1e6bd9b.